packages/cozy-client/src/helpers.js

```diff
@@ -76,6 +78,47 @@ export const generateWebLink = ({
   return url.toString()
 }
 
+/**
+ * Deconstruct the given link in order to retrieve useful data like Cozy's name, domain, or slug
+ *
+ * The given link MUST contain a slug
+ *
+ * @param {string} webLink - link to deconstruct. It should be a link from a Cozy and containing a slug
+ * @param {SubdomainType} subDomainType - whether the cozy is using flat or nested subdomains. Defaults to flat.
```

Comment:
```suggestion
 * @param {SubdomainType} [subDomainType=flat] - whether the cozy is using flat or nested subdomains. 
```

Reply:
https://jsdoc.app/tags-param.html

Reply:
Fixed in https://github.com/cozy/cozy-client/compare/637711a2193c5077ee2ada1cb7eec31c843ee6fc..f14039dd1b109d01d61f354cbd854fdb7804edf8

Thanks